app/src/main/res/values/dimens.xml

```diff
@@ -574,4 +574,32 @@
   <dimen name="profile_reset_pin_activity_profile_reset_input_confirm_pin_margin_end">28dp</dimen>
   <dimen name="profile_reset_pin_activity_profile_reset_save_button_margin_end">28dp</dimen>
 
+  <!-- Profile Edit Fragment -->
+  <dimen name="profile_edit_fragment_view_padding">0dp</dimen>
+  <dimen name="profile_edit_fragment_profile_width_and_height">90dp</dimen>
+  <dimen name="profile_edit_fragment_margin_top_and_start">28dp</dimen>
+  <dimen name="elevation_of_2dp">2dp</dimen>
```

Comment:
Rename this to `profile_edit_fragment_profile_image_elevation` and use it in all xml files.